datasets/ethics/ethics.py

```diff
@@ -0,0 +1,169 @@
+"""ETHICS: Justice Dataset."""
+
+
+import csv
+import os
+
+import datasets
```

Comment:
```suggestion
import datasets```suggestion
import datasets
```

```